frontend/src/lib/k8s/apiProxy/index.ts

```diff
@@ -0,0 +1,67 @@
+/*
```

Comment:
I wonder about git history for apiProxy. Sometimes tracking doesn't work well if lots of the file has changed. If it's not working, doing a move in a separate commit can get around it.

Reply:
moved apiProxy file to apiProxy/index.ts in a separate commit and history seems to be working now